config/tools/config_generator.hpp

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2022 MIT Digital Currency Initiative,
+//                    Federal Reserve Bank of Boston
+//                    MITRE Corporation
```

Comment:
It's probably time for us to split out the Copyright holders into an `AUTHORS` file. It's fine to leave these lines, but I'll work on splitting them out in the next little while.